include/FadeButton.h

```diff
@@ -53,13 +56,16 @@ public slots:
 
 private:
 	QTime m_stateTimer;
+	QTime m_releaseTimer;
+
 	QColor m_normalColor;
 	QColor m_activatedColor;
+	QColor m_holdColor;
```

Comment:
Would help if you'll describe m_holdColor (as opposed to m_activatedColor).

Reply:
Done.